client/dashboard/summary-button/types.ts

```diff
@@ -0,0 +1,74 @@
+type Density = 'low' | 'medium' | 'high';
```

Comment:
Is there really a need to separate the "types" of a single component props to a separate file?

Reply:
No strong opinion here, just asking.

Reply:
There is no need of course. I just followed the same practice with core components and want to see later if we can auto generate the docs from it. It might be simpler with a separate file. It's minor though.